compatibility_lib/compatibility_lib/configs.py

```diff
@@ -12,7 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Common configs for compatibility_lib."""
+"""Common configs for compatibility_lib and compatibility_server.
+
+Note that a unit test exists for checking that the configs.py file in
+compatibility_lib is the same as the configs.py file in compatibility_server.
+
+The reason for this set up is that these modules need to be isolated from
+each other, but there also needs to be consistency in the objects and data
+in this file since they exist in the same workflow.
+"""
+
+
+import enum
+
+
+class PackageStatus(enum.Enum):
+    """Represents a package's compatibility status.
+
+    The status is based on the results of running 'pip install' and
+    'pip check' on the compatibility server
+
+    UNKNOWN_PACKAGE: package not in whitelist
+    INTERNAL_ERROR: unexpected internal error
+    MISSING_DATA: missing package data from package store
+    SELF_INCOMPATIBLE: pip error when installing self
+    INCOMPATIBLE: pip error when installed with another package
+    OUTDATED_DEPENDENCY: package has an outdated dependency
+    SUCCESS: No issues
+    """
+    UNKNOWN_PACKAGE = 'lightgrey'
```

Comment:
Using the color as the value is odd. I'd have another data structure like:

PACKAGE_STATUS_TO_COLOR = {
  PackageStatus.UNKNOWN_PACKAGE: '''
}

Reply:
done